src/package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "ava": "^5.1.0",
     "c8": "^7.12.0",
+    "quibble": "^0.6.15",
```

Comment:
I see `quibble` added but not used – is it safe to remove?

Reply:
Hm, that's a dependency of `testdouble` (`quibble` is a module loader, it's a lot like `esmock`, but these aren't very useful without a mocking library like `testdouble` or similar).

I don't think it needs to be in the `package.json` though, this is probably a remnant of when I was exploring using `esmock` and then tried `quibble` directly, and then realized `testdouble` was closer to what we needed to replace `jest`. I'll see if I can remove it from here and just have it as a dep of td in `package-lock.json`